metadata.json

```diff
@@ -45,6 +45,12 @@
       "operatingsystemrelease": [
         "12"
       ]
+    },
+    {
+      "operatingsystem": "FreeBSD"
```

Comment:
mhm, sorry to nitpick here :( Can you add the Non-EOL version where this is supposed to work? (same for NetBSD)

Reply:
i just went with what Gentoo does ;)